src/Module/Item/Display.php

```diff
@@ -165,12 +165,13 @@ protected function content(array $request = []): string
 	 */
 	protected function displaySidebar(array $item)
 	{
+		$author = [];
 		$shared = $this->contentItem->getSharedPost($item, ['author-link']);
-		if (!empty($shared) && empty($shared['comment'])) {
+		if (count($shared) > 0 && !array_key_exists('comment', $shared)) {
```

Comment:
This is the wrong condition. If `$shared` has keys, `$shared['comment']` will always be defined, but its value can be an empty string.
```suggestion
		if (array_key_exists('comment', $shared) && !$shared['comment']) {
```